patches/brave/third_party/bitcoin-core/src/src-bech32.cpp.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/src/bech32.cpp b/src/bech32.cpp
+index ba3c419d8b6860a69daa07a53efce8698fc4a86d..b3413fd46a0bafac2a147afe385b8947b68bb241 100644
+--- a/src/bech32.cpp
++++ b/src/bech32.cpp
+@@ -374,7 +374,7 @@ DecodeResult Decode(const std::string& str) {
+     std::vector<int> errors;
+     if (!CheckCharacters(str, errors)) return {};
+     size_t pos = str.rfind('1');
+-    if (str.size() > 90 || pos == str.npos || pos == 0 || pos + 7 > str.size()) {
++    if (pos == str.npos || pos == 0 || pos + 7 > str.size()) {
```

Comment:
Here's a link to the slack discussion about this patch and considering alternative options: https://bravesoftware.slack.com/archives/C7VLGSR55/p1703593681208849

We're removing this check because there's times where the bech32 encoding may end up longer in zcash and being more permissive here in bitcoin won't break compatibility there